README.md

```diff
@@ -27,6 +27,15 @@ from this repo, do a
 poetry install
 ```
 
+If Poetry is not already installed, follow these instructions:
+- create a virual environment ([venv](https://docs.python.org/3/library/venv.html#creating-virtual-environments) or [conda](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-with-commands))
+- run `pipx install poetry`
```

Comment:
you may want to add a step stating to activate the environment first before installing poetry

Reply:
good point, will do!

Reply:
Updated in this [commit](https://github.com/monarch-initiative/mondolib/pull/9/commits/a0058b2961c6ea3e64f32089ecf271ea83ef2804)